### PR TITLE
boards/arduino-nano: port arduino nano board

### PR DESCRIPTION
**Pull request by lebrush**

### CODE REVIEW

File: boards/arduino-nano/include/board.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2017   Víctor Ariño
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_arduino-uno Arduino Nano
```
Comment (smlng) [inline boards/arduino-nano/include/board.h]: /uno/nano/

**Comment by lebrush** [comment]
Thanks for the super quick review @smlng. 
It's in fact a very good point, but until a solution is provided this is the only way to support a new board...

**Comment by lebrush** [comment]
True, unfortunately, how the atmega boards are supported nowadays, everything is handled in the `atmega-atmega-common` folder.

**Comment by sebastiankliem** [comment]
I've used your port for an university project and recognized that some referenced directories have changed. I fixed this in this commit: https://github.com/sebastiankliem/RIOT/commit/77d1b53141d1cdf8f73f617941393b17d22c6113

**Comment by smlng** [comment]
@lebrush can you quickly rebase and adapt, I would merge this for release 2018.01 and if possible.

**Comment by SemjonKerner** [comment]
Hi @lebrush, i am very interested in using the nano with riot, are you going to adapt the requested changes soon?

**Comment by maribu** [comment]
As this PR is apparently dead, I created [a new PR](https://github.com/RIOT-OS/RIOT/pull/9499) providing support for the Arduino Nano

**Comment by PeterKietzmann** [comment]
@maribu I guess it's ok after that time. Closing this PR in favor of #9499. Tanks @lebrush anyway!

**Pull request closed without merging**